db/migrate/20180413193115_require_title_person.rb

```diff
@@ -0,0 +1,6 @@
+class RequireTitlePerson < ActiveRecord::Migration
+  def change
+    change_column_null :people, :title, false, "Recruit"
+    change_column_null :people, :title_order, false, 30
```

Comment:
And this to 100, if you agree with the comment above

Reply:
Yes